src/views/check/error-redirect.html

```
@@ -38,7 +42,11 @@
       </ul>
 
       <p class="govuk-body">Contact your IT team for support if you need it.</p>
-    {% elif errorMessage == "The selected file must be a CSV, GeoJSON, GML or GeoPackage file" %}
+    {% elif errorCode == "403" or errorCode == 403 %}
+      <p class="govuk-body" id="bad-upload" tabindex="-1">You must host the URL on a server which does not block access due to set permissions.</p>
```

HTMLHint (1.9.2)

[error] The id value [ bad-upload ] must be unique.

(id-unique)